tests/Generator.Tests/IncrementalGeneratorTests.cs

```diff
@@ -106,10 +106,17 @@ public void CheckGeneratorIsIncremental(
         driver = driver.RunGenerators(compilation);
 
         var result = driver.GetRunResult().Results.Single();
-        var sourceOutputs =
-            result.TrackedOutputSteps.SelectMany(outputStep => outputStep.Value).SelectMany(output => output.Outputs);
-        var (value, reason) = Assert.Single(sourceOutputs);

```

Comment:
This was failing because there are now 2 `context.RegisterSourceOutput`'s (for the mapping diagnostics and method results). That's why I check for the output length `2` now.